collect_app/src/main/java/org/odk/collect/android/widgets/BaseImageWidget.java

```diff
@@ -172,6 +178,19 @@ public void onClick(View v) {
         }
     }
 
+    private File addDefaultImageIfExists(File f) {
```

Comment:
Why do you pass in the File object here? I'd expect this method to just return a File...

Reply:
Because if a default media file doesn't exist too I would like to receive the original file, the one I pass here (not null) in order to display an appropriate message (which file is missing).